core/src/main/java/tech/pegasys/web3signer/core/FilecoinRunner.java

```diff
@@ -71,6 +73,10 @@ protected Router populateRouter(final Context context) {
         context.getArtifactSignerProvider(),
         RELOAD.name(),
         context.getErrorHandler());
+    registerHealthCheck(
+        context.getRouterBuilder(),
+        context.getErrorHandler(),
+        new HealthcheckHandler(Optional.empty()));
```

Comment:
Similar concern to the eth1 healthpoint. Is there anything useful this does for filecoin. Think this is just a upcheck without the db health check?